src/model.d.ts

```diff
@@ -617,23 +665,32 @@ export type WhereAttributeHashValue<AttributeType> =
  */
 export interface IncludeThroughOptions extends Filterable<any>, Projectable {
   /**
-   * The alias of the relation, in case the model you want to eagerly load is aliassed. For `hasOne` /
-   * `belongsTo`, this should be the singular name, and for `hasMany`, it should be the plural
+   * The alias for the join model, in case you want to give it a different name than the default one.
    */
   as?: string;
 
   /**
    * If true, only non-deleted records will be returned from the join table.
    * If false, both deleted and non-deleted records will be returned.
    * Only applies if through model is paranoid.
+   *
+   * @default true
```

Comment:
Isn't it by default turned off?

Reply:
This option is on by default but it also needs paranoid to be on to do anything